vcsh

```diff
@@ -276,6 +276,13 @@ list() {
 	done
 }
 
+list_where_remote() {
+	for VCSH_REPO_NAME in $(list); do
+		GIT_DIR=$VCSH_REPO_D/$VCSH_REPO_NAME.git; export GIT_DIR
+		[ -n "$(git remote)" ] && echo "$VCSH_REPO_NAME"
```

Comment:
```suggestion
		git config branch.$VCSH_BRANCH.remote > /dev/null && echo "$VCSH_REPO_NAME"
```

How about this to check if there is a remote for the branch VCSH thinks it is operating on? This way we can also use the return code of `git config` rather than checking whether STDOUT exists (which by the way might leak expected bits of STDERR to the user).